### ISSUE TITLE
Database of CIs functionality + community building

### ISSUE THREAD

**Issue by rdotsch** [rdotsch/rcicr#22]
Quick thought for new feature, will elaborate later:
- Add functionality to save a list of CIs to some kind of database format which can be shared in an online repository (e.g., `saveCIDatabase()`, which contains the ci list and the original base image).
- Add functionality to load the database (`loadCIDatabase()`)  and do two things: First, generate new stimuli on the basis of the same base image (`generateDatabaseStimuli()`) and second,  when generating CIs, the visual similarities with each of the CIs in the database are reported (e.g., pixel correlations, euclidean distances). 

The point is that if one collects data for several dimensions, say trustworthiness and dominance, these can be saved to a database and shared with others. Other people can then quickly use these databases to find out how trustworthy and dominant the CIs are based on data they collected themselves. 

By collecting many such databases in a single repository, we can create one giant database that computes all of these indices for CIs and which should be straightforward to update from the rcicr package, for instance with `updateCIDatabase()` which should pull the giant database from a server.

In a nutshell, this would crowdsource the creation of a very useful database.
